lib/src/rules/package_prefixed_library_names.dart

```diff
@@ -84,28 +73,33 @@ class _Visitor extends SimpleAstVisitor<void> {
   _Visitor(this.rule);
 
   @override
+  // ignore: prefer_expression_function_bodies
```

Comment:
It seems wrong to recommend using an expression function body when there's no value to return. Somehow `void f() => null;` feels wrong.